### PR TITLE
Dockerfile and Documentation for RHOSP13

### PR DESCRIPTION
**Pull request by yong-huang**
Major changes:

* Add Dell EMC customized Manila share docker file
* Update Cinder VNX document
* Add Cinder Unity document
* Add Manila VNX document
* Add Manila Unity document

### CODE REVIEW

File: cinder/README.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+# Dell EMC Cinder driver containerization with OSP13
+
+## Overview
+
+Dell EMC Cinder driver integrates with the RedHat OpenStack Platform since OSP 6. In OSP13, all Cinder services are able to run within containers.
```
Comment (amagicsun) [inline cinder/README.md]: It is not necessary to add this description. I would suggest to remove this line.

File: cinder/README.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  CinderDellEMCVNXStoragePoolName: ''
+  CinderDellEMCVNXDefaultTimeout: 3600
+  CinderDellEMCVNXMaxLunsPerStorageGroup: 255
+  CinderDellEMCVNXInitiatorAutoRegistration: 'true'
+  CinderDellEMCVNXAuthType: 'global'
+  CinderDellEMCVNXStorageSecurityFileDir: ''
+  CinderDellEMCVNXNaviSecCliPath: '/opt/Navisphere/bin/naviseccli'
+```
+
+For a full detailed instruction of options, please refer to [VNX backend configuration](https://docs.openstack.org/cinder/queens/configuration/block-storage/drivers/dell-emc-vnx-driver.html)
```
Comment (amagicsun) [inline cinder/README.md]: Shouldn't we provide the latest doc: https://docs.openstack.org/cinder/latest/configuration/block-storage/drivers/dell-emc-vnx-driver.html

File: cinder/README.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+  CinderDellEMCUnityBackendName: 'tripleo_dellemc_unity'
+  CinderDellEMCUnitySanIp: '192.168.1.50'
+  CinderDellEMCUnitySanLogin: 'admin'
+  CinderDellEMCUnitySanPassword: 'password'
+  CinderDellEMCUnityStorageProtocol: 'iSCSI'
+  CinderDellEMCUnityIoPorts: ''
+  CinderDellEMCUnityStoragePoolNames: ''
+```
+
+For a full detailed instruction of options, please refer to [Unity backend configuration](https://docs.openstack.org/cinder/queens/configuration/block-storage/drivers/dell-emc-unity-driver.html)
```
Comment (amagicsun) [inline cinder/README.md]: The same as the previous one.
https://docs.openstack.org/cinder/latest/configuration/block-storage/drivers/dell-emc-unity-driver.html

File: manila/README.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
+# Dell EMC Manila driver containerization with OSP13
+
+## Overview
+
+Dell EMC Manila driver integrates with the RedHat OpenStack Platform since OSP 6. In OSP13, all Manila services are able to run within containers.
```
Comment (amagicsun) [inline manila/README.md]: The same comment as the first one.

File: manila/README.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+This instruction provides detailed steps on how to enable the containerization of VNX and Unity Manila driver on top of the OSP Manila images.
+
+The Dell EMC Manila container image contains following RPM packages:
+
+- python-storops
+- python-persist-queue
+- python-cachez
+- python-retryz
+- EMC Naviseccli
```
Comment (amagicsun) [inline manila/README.md]: We don't need naviseccli for manila driver, do we?

File: manila/README.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+  ManilaVNXDriverHandlesShareServers: true
+  ManilaVNXNasLogin: 'admin'
+  ManilaVNXNasPassword: 'password'
+  ManilaVNXNasServer: '192.168.1.49'
+  ManilaVNXServerContainer: 'server_2'
+  ManilaVNXShareDataPools: ''
+  ManilaVNXEthernetPorts: 'cge-2-0'
+```
+
+For a full detailed instruction of options, please refer to [VNX backend configuration](https://docs.openstack.org/manila/queens/configuration/shared-file-systems/drivers/dell-emc-vnx-driver.html)
```
Comment (amagicsun) [inline manila/README.md]: https://docs.openstack.org/manila/latest/configuration/shared-file-systems/drivers/dell-emc-vnx-driver.html

File: manila/README.md [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+  ManilaUnityNasPassword: 'password'
+  ManilaUnityNasServer: '192.168.1.49'
+  ManilaUnityServerMetaPool: 'pool_1'
+  ManilaUnityShareDataPools: ''
+  ManilaUnityEthernetPorts: ''
+  ManilaUnityEmcSslCertVerify: false
+  ManilaUnityEmcSslCertPath: ''
+```
+
+For a full detailed instruction of options, please refer to [Unity backend configuration](https://docs.openstack.org/manila/queens/configuration/shared-file-systems/drivers/dell-emc-unity-driver.html)
```
Comment (amagicsun) [inline manila/README.md]: https://docs.openstack.org/manila/latest/configuration/shared-file-systems/drivers/dell-emc-unity-driver.html

File: cinder/README.md (hunk 1 above)
Comment (yong-huang) [reply cinder/README.md]: Done

File: cinder/README.md (hunk 2 above)
Comment (yong-huang) [reply cinder/README.md]: Done

File: cinder/README.md (hunk 3 above)
Comment (yong-huang) [reply cinder/README.md]: Done

File: manila/README.md (hunk 4 above)
Comment (yong-huang) [reply manila/README.md]: Done

File: manila/README.md (hunk 5 above)
Comment (yong-huang) [reply manila/README.md]: No, we don't, removed this line.

File: manila/README.md (hunk 6 above)
Comment (yong-huang) [reply manila/README.md]: Done

File: manila/README.md (hunk 7 above)
Comment (yong-huang) [reply manila/README.md]: Done

**Pull request merged**